tools/gen_struct_info.py

```diff
@@ -396,7 +396,7 @@ def inspect_code(headers, cpp_opts, structs, defines):
   # -Oz optimizes enough to avoid warnings on code size/num locals
   cmd = [shared.EMCC] + cpp_opts + ['-o', js_file[1], src_file[1],
                                     '-O0', '--js-opts', '0', '--memory-init-file', '0',
-                                    '-Werror', '-Wno-format',
+                                    '-Werror', '-Wno-format', '-Wno-deprecated',
```

Comment:
Oh, this uses the literal form - so we'll need to change that eventually?

Reply:
No, i does not use it directly, but we have tests the verify the EM_CONFIG in the environment works, and that everything that emcc runs internally too.

Reply:
So this change is just for testing purposes? Maybe a comment then, so it is obvious why it's there and when we can remove it?